WooCommerce/WooCommerceTests/Authentication/AuthenticationManagerTests.swift

```diff
@@ -191,6 +191,24 @@ final class AuthenticationManagerTests: XCTestCase {
         XCTAssertTrue(rootController is ULErrorViewController)
     }
 
+    func test_errorViewController_returns_jetpack_connection_error_if_no_site_matches_the_given_self_hosted_site() {
```

Comment:
Nit: Based on the naming, I think we aren't checking that the returned controller is of jetpack connection error type.  We are only checking that the `controller` is of type `ULErrorViewController`. Please correct me if I am wrong.